probatus/feature_elimination/feature_elimination.py

```diff
@@ -98,10 +98,14 @@ def __init__(self, clf, step=1, min_features_to_select=1, cv=None, scoring='roc_
                 iteration. It is recommended to use float, since it is faster for a large number of features, and slows
                 down and becomes more precise towards less features. Note: the last round may remove fewer features in
                 order to reach min_features_to_select.
+                If columns_to_keep parameter is specified in the fit method, step is the number of features to remove after 
+                keeping those columns.
 
             min_features_to_select (int, optional):
                 Minimum number of features to be kept. This is a stopping criterion of the feature elimination. By
-                default the process stops when one feature is left.
+                default the process stops when one feature is left. If columns_to_keep is specified in the fit method, 
```

Comment:
maybe we should do 
```
min_features_to_select=min_features_to_select-len(columns_to_keep)
if min_features_to_select<0:
     min_features_to_select=0
```
This way the program is going to stop at only columns_to_keep if the minimum features to select is lower. The benefit of that is the fact that by default the feature set of only columns_to_keep is going to be optimized and evaluated, so the users would know how good is the performance of using only columns_to_keep. What do you think?

Reply:
Simplified the min_features_to_select. So when the input columns_to_keep is provided, the final set is the columns_to_keep. Thus the end users can would know how good is the performance of using only columns_to_keep.